hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OFSPath.java

```diff
@@ -186,6 +189,21 @@ public boolean isVolume() {
     return this.getBucketName().isEmpty() && !this.getVolumeName().isEmpty();
   }
 
+  private static String md5Hex(String input) {
+    try {
+      MessageDigest md = MessageDigest.getInstance("MD5");
+      byte[] digest = md.digest(input.getBytes());
```

Comment:
I have implemented my own md5sum in OFSPath, but the check (findbugs) doesn't like it:
```
Found reliance on default encoding in org.apache.hadoop.fs.ozone.OFSPath.md5Hex(String): String.getBytes()  At OFSPath.java:[line 195]
```

Reply:
Added parameter `StandardCharsets.UTF_8` to `getBytes()`. Shouldn't complain now.

Note: Force pushed to make the extra change into only one extra commit. Easier for manual cherry-picking later.